radio/src/opentx.cpp

```diff
@@ -2572,7 +2572,7 @@ int main()
   // lcdSetRefVolt(25);
 #endif
 
-#if defined(PCBTARANIS) || defined(PCBHORUS)
+#if defined(SPLASH) && (defined(PCBTARANIS) || defined(PCBHORUS))
```

Comment:
ah perhaps I did something wrong with my latest commit? I introduced SPLASH=FRSKY for X7 / X9